packages/ui/components/form/inputs/Input.tsx

```diff
@@ -341,7 +341,3 @@ export function InputGroupBox(props: JSX.IntrinsicElements["div"]) {
     </div>
   );
 }
-
-export const MinutesField = forwardRef<HTMLInputElement, InputFieldProps>(function MinutesField(props, ref) {
```

Comment:
Is this not needed any more?

Reply:
No, the `minimumBookingNotice` was the only place in the codebase this was used